buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterFormationTasks.groovy

```diff
@@ -221,8 +252,8 @@ class ClusterFormationTasks {
         Map esConfig = [
                 'cluster.name'                 : node.clusterName,
                 'pidfile'                      : node.pidFile,
-                'path.repo'                    : "${node.homeDir}/repo",
-                'path.shared_data'             : "${node.homeDir}/../",
+                'path.repo'                    : "${node.baseDir}/../repo",
+                'path.shared_data'             : "${node.baseDir}/../",
```

Comment:
Why does what I suggested not work? Right now I believe baseDir points to build/cluster/nodeN, which means the shared_data path will be build/cluster, which is *underneath* every node dir. I am suggesting it be build/cluster/shared.